src/main/kotlin/pet/project/app/dto/book/ResponseBookDto.kt

```diff
@@ -0,0 +1,10 @@
+package pet.project.app.dto.book
+
+data class ResponseBookDto(
+    val id: String?,
+    val title: String?,
+    val description: String?,
+    val yearOfPublishing: Int?,
+    val amazonPrice: Double?,
+    val amountAvailable: Int?
+)
```

Comment:
```suggestion
)

```
please, fix all places

Reply:
Fixed: added new line at the end of all files